skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+from ..draw import polygon
+
+
+def _mask_from_verts(verts, shape):
+    mask = np.zeros(shape, dtype=bool)
+    pr = [y for x, y in verts]
+    pc = [x for x, y in verts]
+    rr, cc = polygon(pr, pc, shape)
+    mask[rr, cc] = 1
+    return mask
+
+
+def manual(image, alpha=0.4, return_masks=False, draw_lines=False):
```

Comment:
`draw_lines` does not sound good to me.
I think either the user should have a possibility to switch tools (lasso/polygonal) interactively (_this would be just perfect_) or this should be renamed to `mode` or `tool`.